src/datumaro/components/media.py

```diff
@@ -1195,3 +1202,218 @@ def as_dict(self) -> Dict[str, Any]:
             "data": attrs["data_in"],
             "size": attrs["size"],
         }
+
+
+TableDtype = TypeVar("TableDtype", str, int, float)
+
+
+class Table(MediaElement[pd.DataFrame]):
```

Comment:
Is this necessary to be added to `Media`? I feel that we only need `TableRow`.

Reply:
You're right. I'll remove Table from Media